.github/workflows/create-release-draft.yml

```diff
@@ -131,7 +131,7 @@ jobs:
 
       - name: Build docker image
         run: |
-          ./parachain/scripts/build-docker.sh production ${{ env.RELEASE_TAG }}
+          ./parachain/scripts/build-docker.sh production ${{ env.RELEASE_TAG }} --features=on-chain-release-build
```

Comment:
Why not use `metadata-hash` directly?

Reply:
Can we have this enabled by default for both `release` and `production` build? It seems that we are testing one variant in general CI (without this feature) but actually release another variant

I ask this because it can be a breaking change for the worker, see https://github.com/integritee-network/worker/pull/1603

Reply:
> Why not use `metadata-hash` directly?

@Kailai-Wang  How about we just add it all by default without add a new feature? From the [guide](https://paritytech.github.io/polkadot-sdk/master/polkadot_sdk_docs/guides/enable_metadata_hash/index.html), it recommends to separate to save building time. But it actually turns out there is only less than 2 mins difference.

Reply:
Regarding `release` and `prodcution`, I think `release` is for `production`.  Our current `release` is actually used for testing purpose. Do you think it's worth to rename current `release` to some different name?